src/measure_theory/integral/bochner.lean

```diff
@@ -146,6 +146,54 @@ namespace measure_theory
 
 variables {α E F 𝕜 : Type*}
 
+section weighted_smul
+
+variables [normed_group F] [normed_space ℝ F] {m : measurable_space α} {μ : measure α}
+
+/-- Given a set `s`, return the continuous linear map `λ x, (μ s).to_real • x`. The extension of
+that set function through `set_to_L1` gives the Bochner integral of L1 functions. -/
+def weighted_smul {m : measurable_space α} (μ : measure α) (s : set α) : F →L[ℝ] F :=
+{ to_fun := λ x, (μ s).to_real • x, map_add' := by simp, map_smul' := λ c x, by rw smul_comm, }
+
+@[simp] lemma weighted_smul_zero {m : measurable_space α} (μ : measure α) (s : set α) :
+  weighted_smul μ s (0 : F) = 0 :=
+by simp [weighted_smul]
```

Comment:
Do you need this lemma?  Since `continuous_linear_map.map_zero` is a simp-lemma I'd have thought that this special case doesn't need to exist.